src/styles/variables.css

```diff
@@ -5,6 +5,7 @@
   --colony-black: rgb(31, 35, 39);
   --colony-blue: rgb(79, 93, 255);
   --colony-light-blue: rgb(238, 242, 245);
+  --light-blue: rgb(238, 239, 255);
```

Comment:
These 2 colors are a bit different. Should I use '--colony-light-blue' instead of '--light-blue'?
<img width="326" alt="Screenshot 2023-01-04 at 08 22 49" src="https://user-images.githubusercontent.com/91876137/210504164-ab2985a0-4e0d-4f68-9af4-3c82e05a0ba6.png">
<img width="342" alt="Screenshot 2023-01-04 at 08 23 21" src="https://user-images.githubusercontent.com/91876137/210504176-7b72f018-9f4f-43ee-96ce-af2fb2a9ddf0.png">
'--colony-light-blue' is the color below.

Reply:
Yeah, that should be fine, in order to not have to introduce a new color.